docs/source/api/spatial-indexing.rst

```diff
@@ -381,3 +381,60 @@ polyfill
     |[577586652210266111, 578360708396220415, 577269992861466623]|
     +------------------------------------------------------------+
 
+index_to_centroid
+********
+
+.. function:: index_to_centroid(index_id)
```

Comment:
Sure; I do need the functionality for a blog post—but I might just use H3 UDFs to do it and might go back in later to update.

Reply:
@tiems90 We are currently aligning the naming convention to the product H3 one, except that we are using a generic grid instead of the specific H3.
Could you please rename `index_to_centroid` to `grid_centeraswkb`?